cypress/e2e/pages/apply/[resident]/summary.cy.ts

```diff
@@ -50,11 +51,39 @@ describe('Apply resident summary page', () => {
   beforeEach(() => {
     cy.loginAsResident(applicationId, true);
     cy.task('clearNock');
-    cy.mockNotifyEmailResponse();
+  });
+
+  it('succesfully disqualifies application if is not eligible to apply and disqualify email notify action fails', () => {
+    //Application object does not reflect the correct state after patch, but it doesn't matter for this test
+    cy.mockHousingRegisterApiPatchApplication(
+      applicationId,
+      ineligibleApplicationWithCompletedMainApplicantSections,
+      1000,
+      StatusCodes.OK,
+      false
+    );
+
+    cy.mockHousingRegisterApiGetApplications(
+      applicationId,
+      ineligibleApplicationWithCompletedMainApplicantSections,
+      true
+    );
+
+    cy.mockNotifyEmailResponse(StatusCodes.INTERNAL_SERVER_ERROR);
```

Comment:
Is there any user feedback that this failure happened?

Reply:
These are all (notify) currently silent failures. At this point, we're going to leave it that way to be addressed later as a known issue. 

This test was really just to ensure that in the try block a failure of that promise wouldn't stop the next action. As it's not unwrapped it's assumed as much but just wanted to make sure in a test.